Makefile

```diff
@@ -1,30 +1,33 @@
+container_tag ?= rated-log-indexer-indexer
+container_name ?= rated_log_indexer
+
 ##@ 🚀  Getting started
 .PHONY: run
 run: build up ## Ready, set, go!
 
 .PHONY: build
 build:  ## Build the Docker container
-	@docker compose build
+	@docker build -t $(container_tag) .
 
 .PHONY: up
 up: ## Start services
-	@docker compose up -d --force-recreate indexer db redis
+	@docker run -d --rm --name $(container_name) $(container_tag)
 
 .PHONY: down
 down: ## Stop services
-	@docker compose down
+	@docker stop $(container_name)
 
 .PHONY: remove
 remove: ## Remove containers, images, networks, and volumes
-	@docker compose down --rmi all --volumes --remove-orphans
+	@docker remove --volumes $(container_tag) || true
 
 .PHONY: logs
-logs: ## Output container logs. For one specific service use services variable. Example: `make logs services="app"
-	@docker compose logs -f --tail 50
+logs: ## Output container logs.
+	@docker logs --follow --tail 50 $(container_name)
 
 .PHONY: install
 install:  ## Install dependencies in Docker
-	@docker compose run --rm indexer pip install -r requirements.txt
+	@docker run --rm $(container_tag) pip install -r requirements.txt
```

Comment:
I don't really see the point of this now. `make build` will already take care of rebuilding as necessary.

At the same time `make ready` really doesn't prepare the local environment for development or running e.g. the tests. We could repurpose this command to install requirements locally, possibly using `uv` which is really nice.